src/main/scala/com/gu/zuora/model/response/Responses.scala

```diff
@@ -77,20 +77,13 @@ object SubscribeResponseAccount {
   implicit val codec: Codec[SubscribeResponseAccount] = capitalizingCodec
 }
 case class SubscribeResponseAccount(
-  accountNumber: String,
-  subscriptionNumber: String,
-  gatewayResponse: String,
-  paymentId: String,
-  invoiceResult: InvoiceResult,
-  totalTcv: Float,
-  subscriptionId: String,
-  totalMrr: Float,
-  paymentTransactionNumber: String,
-  accountId: String,
-  gatewayResponseCode: String,
-  invoiceNumber: String,
-  invoiceId: String,
-  success: Boolean
+    accountNumber: String,
```

Comment:
this always happens to me as well for some reason, but it was actually correct before!

Reply:
It's Scalariform messing about, I've reverted this change